server/server.js

```diff
@@ -19,6 +20,12 @@ app.use('/api/problems', problemsRouter);
 
 app.use(Express.static(root, {maxAge: '2d'}));
 app.use(fallback('index.html', {root}));
+app.use((req, res, next) => {
```

Comment:
remove this CORS resolution - this issue should be fixed on webpack-dev-server side:
just configure proxy for dev server:
https://webpack.js.org/configuration/dev-server/#devserver-proxy